<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alter</class>
 <widget class="QWidget" name="alter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>108</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
    color: rgb(255, 255, 255);
    background-color: rgb(167, 205, 255);
    border:ridge;
    padding: 3px;
    font-family: &quot;Verdana&quot;;
    font-size: 10px;
    text-align: center left;
padding-left: 3px;
}
QPushButton:hover, QPushButton:pressed , QPushButton:checked
{
    background-color: rgb(85, 170, 255);
    padding-left: 10px;
    font-weight:200
}
QPushButton:hover
{
    background-image: url(://1);
    background-repeat:repeat;
    background-position: right;
}
QPushButton:pressed, QPushButton:checked
{
    background-image: url(://2);
    background-repeat:no-repeat;
    background-position: center left;
}</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GO！</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="up_label">
      <property name="font">
       <font>
        <family>华文琥珀</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>  Label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="up_button1">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Studing</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="up_button2">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eating</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="up_button3">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>just thinking</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="up_box">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtTop</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="down_label">
      <property name="font">
       <font>
        <family>华文琥珀</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>   item</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="down_button1">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Math</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="down_button2">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Coding</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="down_button3">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Engilish</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="down_box">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
